tests/export_test.py

```diff
@@ -1369,6 +1370,64 @@ def test_multi_platform_nested_inside_single_platform_export(self):
     res2 = exp2.call(x)
     self.assertAllClose(res2, _testing_multi_platform_fun_expected(x))
 
+  def test_multi_platform_mlir_lower_fun_with_platform_specific_primitives(self):
```

Comment:
You need to check that the capability is at least 8.0 as in e.g. https://github.com/google/jax/blob/1c06114de2652ae6768e6942fb34b373b97b948b/tests/pallas/pallas_test.py#L133.